redash/models/__init__.py

```diff
@@ -64,7 +65,7 @@ def get(self, query_id):
 
 
 @python_2_unicode_compatible
-class TableMetadata(db.Model):
+class TableMetadata(TimestampMixin, db.Model):
     id = Column(db.Integer, primary_key=True)
```

Comment:
@emtwo This will require an update to the migration file.

Reply:
oops! Thank you! I've updated the migration file